packages/view-util/lib/viewGraph.js

```diff
@@ -0,0 +1,40 @@
+import * as ns from '@view-builder/core/ns.js'
+import { hydra, rdf } from '@tpluscode/rdf-ns-builders'
+import $rdf from 'rdf-ext'
+import TermMap from '@rdfjs/term-map'
+
+const viewBuilderNs = ns.viewBuilder().value
+const hydraNs = hydra().value
+
+export function removeApiProperties({ predicate }) {
+  const url = predicate.value
+
+  return !url.startsWith(viewBuilderNs) && !url.startsWith(hydraNs)
+}
+
+export function sourcesToBlankNodes(dataset) {
+  function isSource(term) {
+    return dataset.match(term, rdf.type, ns.view.CubeSource).size > 0
```

Comment:
I wonder if just CubeSources need to be blanks, 
I'm saying this because rdf-cube-view-query is using describe to retrieve the views. https://github.com/zazuko/rdf-cube-view-query/blob/3d088e6c634fe8e251ad3f6be483a7852996e9b7/lib/View.js#L327

Reply:
I was specific here because in view builder only the sources are actually named nodes. Everything else which is needs to be blank is already blank

Reply:
Ok!

Reply:
I also just changed this to look for `view:source` in case the type is not explicit